addons/event/views/event_slot_views.xml

```diff
@@ -65,8 +70,8 @@
         <field name="name">Slots</field>
         <field name="res_model">event.slot</field>
         <field name="view_mode">calendar,list,form</field>
+        <field name="mobile_view_mode">list</field>
         <field name="domain">[('event_id', '=', active_id)]</field>
-        <field name="context">{'default_event_id': active_id}</field>
```

Comment:
This action becomes the default action for slots right? We call it indirectly from `action_open_slot_calendar`. We could rename it since we already have an upgrade ?